### PR TITLE
Added Bosnian provider

### PR DESCRIPTION
**Pull request by mirzap**
Signed-off-by: Mirza Pasic mirza.pasic@bosnadev.com

### CODE REVIEW

File: src/Faker/Provider/bs_BA/Address.php [hunk 1]
```diff
@@ -0,0 +1,319 @@
+<?php
+
+namespace Faker\Provider\bs_BA;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $streets = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/bs_BA/Address.php]: Can you add a datasource? Can you order alphabetically, one line per first letter?

File: src/Faker/Provider/bs_BA/Address.php [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+        '{{cityName}}',
+    );
+
+    protected static $postcode = array('#####');
+
+    /**
+     * @link https://bs.wikipedia.org/wiki/Spisak_gradova_i_op%C4%87ina_u_Bosni_i_Hercegovini
+     */
+    protected static $cityNames = array(
+        "Sarajevo", "Banovići", "Bihać", "Bosanska Krupa", "Bosanski Petrovac", "Bosansko Grahovo", "Breza", "Bugojno", "Busovača",
```
Comment (fzaninotto) [inline src/Faker/Provider/bs_BA/Address.php]: Can you order alphabetically, one line per first letter?

File: src/Faker/Provider/bs_BA/Person.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Faker\Provider\bs_BA;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link http://www.just-think-it.com/sbn/bosnianm.htm
```
Comment (fzaninotto) [inline src/Faker/Provider/bs_BA/Person.php]: This source doesn't have a Licence, and I doubt we can reuse the data in a MIT License project.

File: src/Faker/Provider/bs_BA/Person.php [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        "Nihada", "Nikolina", "Nina", "Nives", "Nudžejma", "Petra", "Rahima", "Ranka", "Rejhana", "Rifeta", "Šejla", "Šemsa",
+        "Sabina", "Sadika", "Sajra", "Saliha", "Samija", "Samira", "Samra", "Sandra", "Sanela", "Sanija", "Sara", "Sarah",
+        "Selma", "Selmira", "Semija", "Semira", "Senada", "Senija", "Senka", "Sinolicka", "Sladjana", "Sonja", "Spomenka",
+        "Stojanka", "Suada", "Sumeja", "Suvada", "Taiba", "Tajra", "Tamara", "Tanja", "Tatjana", "Tihana", "Tijana", "Una",
+        "Valentina", "Vanja", "Vasvija", "Vesna", "Vildana", "Zana", "Željka", "Zana", "Zara", "Zdenka", "Zejneba",
+        "Zerina", "Zijada", "Zlata", "Zlatka", "Zorana", "Zorica", "Zuhra", "Zulejha", "Zumreta"
+     );
+
+    /**
+     * @link http://surnames.behindthename.com/submit/names/usage/bosnian
```
Comment (fzaninotto) [inline src/Faker/Provider/bs_BA/Person.php]: Copyrighted contend.

http://surnames.behindthename.com/info/copyright

Comment (mirzap) [inline src/Faker/Provider/bs_BA/Person.php]: I already filtered the list to reflect real Bosnian names. The list of names is provided by Federal Bureau of Statistics and it's in public domain. I'll update the source accordingly.

**Comment by fzaninotto** [comment]
No feedback, closing.

**Pull request closed without merging**